legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/SQLQueryToString.pure

```diff
@@ -136,6 +137,7 @@ Class meta::relational::functions::sqlQueryToString::DbExtension
    tableNameToIdentifier: meta::pure::metamodel::function::Function<{String[1], DbConfig[1] -> String[1]}>[0..1];
    columnNameToIdentifier: meta::pure::metamodel::function::Function<{String[1], DbConfig[1] -> String[1]}>[0..1];
    identifierProcessor: meta::pure::metamodel::function::Function<{String[1], DbConfig[1] -> String[1]}>[1];
+   relationalDDLCommandsProvider : meta::relational::functions::sqlQueryToString::RelationalDDLCommandsProvider[0..1];
```

Comment:
just write RelationalDDLCommandsProvider, drop the full path meta::relational::functions::sqlQueryToString::, its un-necessary